CSS `font-size` parameter hyper-sensitive upgrading from 3.0.0 to 3.0.1/3.0.2

<!-- IMPORTANT
Bug reports that do not make an effort to help the developers will be closed without notice.
Make sure that this bug has not already been opened and/or closed by searching the issues on GitHub, as duplicate bug reports will be closed.
A bug report simply stating that Darktable crashes is unhelpful, so please fill in most of the items below and provide detailed information.
-->

**Describe the bug**

When trying to adjust the basic font size of the interface, editing `/usr/share/darktable/themes/darktable.css`, changing the value from the default 1em to 0.9em or 1.1em creates very big changes in font size (see the screenshots). It worked perfectly in 3.0.0.  (**corrected, it was 3.0.1 in the first version**)

**To Reproduce**

1. Edit  /usr/share/darktable/themes/darktable.css and around line 160 change the value of `font-size:` from `1em` to something like `0.9em` or `1.1em`.
2. Launch `darktable`

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
A change of 10% of that parameter should be reflected as a change of (around) 10% of the font size in the main interface. 
It worked that way on 3.0.0

**Screenshots**

Here I made two screenshots with two values of `font-size:`

![2020-04-18_22-35](https://user-images.githubusercontent.com/6414907/79683117-0d337400-8228-11ea-9591-f745f973925d.png)

![2020-04-18_22-36](https://user-images.githubusercontent.com/6414907/79683119-0efd3780-8228-11ea-8460-d33bf7c2b720.png)


**Platform (please complete the following information):**
 - Darktable Version: 3.0.2, installed by PPA https://launchpad.net/~ubuntuhandbook1/+archive/ubuntu/darktable
 - OS: Ubuntu 18.04
 - No OpenCL
 - Internal Intel graphic card

**Additional context**

I can use a workaround changing the DPI of the monitor (although it shouldn't be needed; in 3.0.1 the automatic detection worked fine.)

BTW, thank you again for this great SW

@Nilvus  ok, I reduced it a lot but I am stuck. My `git-bisect` arrived here: 

![image](https://user-images.githubusercontent.com/6414907/80239440-8b48ae00-8660-11ea-91b4-c65c89d8f0d9.png)

...but the problem now is that 18404309fa0356b4245a8d843bff6babefa2cf34 doen not compile, gives me an error:

```
[ 67%] Building C object src/iop/CMakeFiles/vibrance.dir/introspection_vibrance.c.o
/home/romano/software/darktable/darktable/build/src/iop/introspection_colorout.c: In function ‘introspection_init’:
/home/romano/software/darktable/darktable/build/src/iop/introspection_colorout.c:179:35: error: ‘DT_COLORSPACE_PQ_REC2020’ undeclared (first use in this function); did you mean ‘DT_COLORSPACE_LIN_REC2020’?
     { "DT_COLORSPACE_PQ_REC2020", DT_COLORSPACE_PQ_REC2020 },
                                   ^~~~~~~~~~~~~~~~~~~~~~~~
                                   DT_COLORSPACE_LIN_REC2020
```

Hmmm... let's see if I can exit from the hole here...

--- a/data/themes/darktable.css
+++ b/data/themes/darktable.css
@@ -223,6 +223,7 @@ eventbox *,
 box,
 box *
 {
+  font-size: 1em;
   background-color: transparent;
 }
 

